ajax/libs/rowGrid/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "rowGrid",
+  "description": "A small, lightweight JavaScript plugin for placing items in straight rows",
+  "filename": "jquery.row-grid.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/brunjo/rowGrid.js.git"
+  },
+  "keywords": [
+    "jQuery",
+    "grid",
+    "layout",
+    "images"
+  ],
+  "author": "Bruno Joseph <bruno.joseph95@gmail.com> (http://www.brunojoseph.com/)",
+  "license": "MIT",
+  "homepage": "https://brunjo.github.io/rowGrid.js/",
+  "npmName": "rowGrid.js",
```

Comment:
Let's use git auto-update instead for more complete versions:
```
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/brunjo/rowGrid.js.git",
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "jquery.row-grid*"
        ]
      }
    ]
  }
```
(Remove `npmName` and `npmFileMap`)